app/assets/stylesheets/utilities/_typography.scss

```diff
@@ -99,3 +99,8 @@ h6,
 .h6 {
   @extend %h6;
 }
+
+.text-center--line {
+  border-bottom-color: color('primary-light');
+  line-height: 0.1em;
```

Comment:
I'm observing that at small phone sizes (e.g. iPhone SE) where text wrapping occurs, the small line-height causes the text to collapse on itself in French and Spanish versions.

![image](https://user-images.githubusercontent.com/1779930/191348529-accdc5b3-a977-4008-a770-af3b79e5d39d.png)

![image](https://user-images.githubusercontent.com/1779930/191348619-7cee6b65-2b2d-40b4-87a5-d2399a919e20.png)